admin/services/pin.service.js

```diff
@@ -51,4 +51,25 @@ pinService.expirePupilPin = async (token, checkCode) => {
   }
 }
 
+/**
+ * Check and expire pupil pins based on set of provided pupil Ids
+ * @param pupilIds
+ */
+
+pinService.expireMultiplePins = async (pupilIds) => {
+  let pupils = []
+  for (let index = 0; index < pupilIds.length; index++) {
+    const id = pupilIds[ index ]
+    const pupil = await pupilDataService.findOne({_id: ObjectId(id)})
+    if (pupil.pin || pupil.pinExpiresAt) pupils.push(pupil)
+  }
+  if (pupils.length === 0) return
+  pupils = pupils.map(p => {
+    p.pin = null
+    p.pinExpiresAt = null
+    return p
+  })
+  await pupilDataService.updateMultiple(pupils)
```

Comment:
Should this not be simply ```return pupilDataService.updateMultiple(pupils)``` - as we always want a function to return something? - I'm looking at ESLint for guidance:
https://eslint.org/docs/rules/no-return-await

Reply:
The difference here is that the caller does not need to capture anything from this method.
The rule you linked is relevant to `return await` which I don't need to do anyway as the method will be behaving like a void.

Reply:
Yes `return pupilDataService.updateMultiple(pupils)` is correct

Reply:
@jon-shipley @pris54 even though as I said the caller does not need it, I will change the await to return to avoid confusion